### PR TITLE
Add Gilnean Mastiff Spell for Worgen Hunters

### PR DESCRIPTION
**Pull request by Cosmodor**

### CODE REVIEW

**Comment by saqirmdev** [comment]
Typo 

(22,3,96234, 'Gilnean Mastiff")"';

**Comment by Cosmodor** [comment]
typos are my nemesi XD

File: updates/022_2013_02_28_00_world_playercreateinfo_spell.sql [hunk 1]
```diff
@@ -11,3 +11,6 @@ INSERT INTO `playercreateinfo_spell` (`race`,`class`,`Spell`,`Note`) VALUES
 (10,8,61437, 'Opening'),
 (10,9,61437, 'Opening');
 */
+/* Add specific worgen spell to allow players to have Gilnean Mastiff when creating a worgen hunter */
+INSERT INTO `playercreateinfo_spell` (`racemask`,`classmask`,`Spell`,`Note`) VALUES
+(22,3,96234, 'Gilnean Mastiff';
```
Comment (ghost) [inline updates/022_2013_02_28_00_world_playercreateinfo_spell.sql:16]: You are missing " ) "

Example: (22,3,96234, 'Gilnean Mastiff');

**Comment by Vincent-Michael** [comment]
use new *.sql for your fix

**Pull request closed without merging**